jsonb-platform-tck/pom.xml

```
@@ -0,0 +1,375 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>tck</artifactId>
+        <groupId>fish.payara.jakarta.tests.tck</groupId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>jsonb-platform-tck-runner</artifactId>
+    <packaging>jar</packaging>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <payara.lib.dir>${payara.home}${file.separator}glassfish${file.separator}lib</payara.lib.dir>
+        <jakarta.tck.arquillian.version>11.0.0</jakarta.tck.arquillian.version>
+        <jakarta.tck.common.version>11.0.0</jakarta.tck.common.version>
+        <ts.home>${project.basedir}${file.separator}jakartaeetck</ts.home>
+        <skipServerStartStop>false</skipServerStartStop>
```

Missing a default javatest-group property

```suggestion
        <skipServerStartStop>false</skipServerStartStop>
        <javatest-testGroups>tck-javatest</javatest-testGroups>
```